.redhat-ci.yml

```diff
@@ -0,0 +1,62 @@
+---
+
+# Example format:
+# https://github.com/jlebon/redhat-ci/blob/master/sample.redhat-ci.yml
+
+# Internal Job (for realtime monitoring):
+# https://hurl.corp.redhat.com/eb0faa3
+
+container:
+    image: docker.io/stackbrew/centos:latest
```

Comment:
ahh interesting.  Naw, it's part of their build-system, has always existed as far as I know.  It's not a big deal, can switch to the shorter form for conformity-sake.